projects/file-parsing/main.go

```diff
@@ -0,0 +1,65 @@
+package main
+
+import (
+	"bytes"
+	filesConverting "file-parsing/types-converting"
+	"fmt"
+	"io"
+	"os"
+	"path/filepath"
+)
+
+func main() {
+	files := []string{"examples/json.txt", "examples/repeated-json.txt", "examples/data.csv", "examples/custom-binary-le.bin", "examples/custom-binary-be.bin"}
+	var result []filesConverting.Score
+
+	for _, file := range files {
+		textByte, err := readingFiles(file)
+		if err != nil {
+			os.Exit(2)
+		}
+		fileNameExtension := filepath.Ext(file)
+
+		switch fileNameExtension {
+		case ".txt":
+			result = filesConverting.JsonToStruct(textByte)
+		case ".csv":
+			result = filesConverting.CsvToStruct(textByte)
+		default:
+			result = filesConverting.CustomBinaryToStruct(textByte)
+		}
+
+		var b bytes.Buffer
+		findMinMax(&b, result)
+	}
+}
+
+func readingFiles(file string) ([]byte, error) {
```

Comment:
I'd generally expect this function to be called `readFile` rather than `readingFiles`:
1. `read` vs `reading` - functions are generally named with with imperative verb forms
2. `file` vs `files` - it only reads one file, so I'd expect the name to be singular.